src/archanaNJun20/RemoveDuplicateNames.java

```diff
@@ -0,0 +1,22 @@
+package archanaNJun20;
+
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class RemoveDuplicateNames {
+	void removeDuplicateNamesFromString(String[] input) {
+		Set<String> nameString = new LinkedHashSet<>();
+		for(String name:input) 
+			nameString.add(name);
+		Iterator<String> iterator = nameString.iterator();
+		while (iterator.hasNext()) {
+			String temp=iterator.next();
+			System.out.println(temp+"->"+temp.length());
+		}
```

Comment:
we have to find and print "Total unique elements are :- 6" as well

Reply:
Code is updated based on your comment.Thank you